app/src/main/java/com/github/h3lp3rs/h3lp/database/FireDatabase.kt

```diff
@@ -9,13 +10,15 @@ import java.util.concurrent.CompletableFuture
 
 /**
  * Implementation of a NoSQL external database based on Firebase
+ * @param path The path to the Firebase database (we use several databases as defined in Databases),
+ * in actuality, they are all on the same Firebase database but are different children on it to
+ * allow us to separate the database into its different uses
  */
-internal class FireDatabase(path : String) : Database {
+internal class FireDatabase(path: String) : Database {
 
-    private val db : DatabaseReference =
-        Firebase.database("https://h3lp-signin-default-rtdb.europe-west1.firebasedatabase.app/").reference.child(
-            path
-        )
+    private val db: DatabaseReference = Firebase
+        .database("https://h3lp-signin-default-rtdb.europe-west1.firebasedatabase.app/")

```

Comment:
Just wondering if we shouldn't put this url in a value files in res folder

Reply:
Done